tests/graphml.rs

```diff
@@ -121,14 +123,9 @@ fn node_and_edge_display_weight() {
     <edge id="e0" source="n0" target="n1">
       <data key="weight">depends on</data>
     </edge>
-  </graph>"#;
-    let expected2 = r#"<key id="weight" for="edge" attr.name="weight" attr.type="string" />"#;
-    let expected3 = r#"<key id="weight" for="node" attr.name="weight" attr.type="string" />"#;
-    let expected4 = r#"</graphml>"#;
+  </graph>
+</graphml>"#;
 
     // HashSet output is unordered, therefore we do not know the order of the keys
     assert!(xml.starts_with(expected1));
-    assert!(xml.contains(expected2));
```

Comment:
The `assert!`s here should be kept as they check that the keys are actually written.

Reply:
oh i see, the hashset is making the output of the keys non deterministic. Not sure where that happens but i'll take your word for it.